packages/arb-avm-cpp/data_storage/include/data_storage/arbcore.hpp

```diff
@@ -178,9 +178,12 @@ class ArbCore {
    public:
     // Logs Cursor interaction
     bool logsCursorRequest(size_t cursor_index, uint256_t count);
-    std::optional<std::vector<value>> logsCursorGetLogs(size_t cursor_index);
+    std::optional<std::vector<value>> logsCursorGetLogs(
```

Comment:
I'm not a fan out using output parameters this way. I think it'd be better to use an `std::pair` or declare a new struct type for this and `logsCursorGetDeletedLogs`

Reply:
fixed